packages/web-react/src/components/Matrix/README.md

```diff
@@ -0,0 +1,117 @@
+# Matrix
+
+The Matrix component distributes nested components in equal columns and allows
+their content to be aligned. It's especially useful for multiple stacked layouts
+where content needs to be organized in a uniform grid pattern.
+
+## Basic Usage
+
+```jsx
+<Matrix>{/* Content items */}</Matrix>
+```
+
+## Custom Spacing
+
+You can use the `spacing` prop to apply custom spacing between items in both horizontal and vertical directions. The prop
+accepts either a spacing token (e.g. `space-100`) or an object with breakpoint keys and spacing token values.
+
+You can set custom spacing in the horizontal (x-axis) and vertical (y-axis) direction separately using the `spacingX` and `spacingY` props.
+
+### Custom Spacing
+
+```jsx
+<Matrix spacing="space-1200">{/* Content items */}</Matrix>
+```
+
+### Custom Responsive Spacing
+
+```jsx
+<Matrix spacing={{ mobile: 'space-400', tablet: 'space-800' }}>{/* Content items */}</Matrix>
+```
+
+### Custom Responsive Horizontal (x-Axis) Spacing
+
+```jsx
+<Matrix spacingX={{ mobile: 'space-400', tablet: 'space-800' }}>{/* Content items */}</Matrix>
+```
+
+### Custom Responsive Vertical (y-Axis) Spacing
+
+```jsx
+<Matrix spacingY={{ mobile: 'space-400', tablet: 'space-800' }}>{/* Content items */}</Matrix>
+```
+
+## Scrollable Matrix
+
+For scrollable Matrix, you can use the `ScrollView` component to wrap the `Matrix` component. This allows the content to be scrolled horizontally.
+
+```jsx
+<ScrollView direction="horizontal" data-spirit-toggle="scrollView">
+  <Matrix>{/* Content items */}</Matrix>
+</ScrollView>
+```
+
+## Responsive Matrix
+
+To create a responsive Matrix, you can use the `cols` prop to define the number of columns for each breakpoint.
+You need to explicitly set the `rows` prop to define the number of Matrix rows for each breakpoint, as well as the `itemRows` prop to specify the number of items per row.
+
+```jsx
+<Matrix
+  cols={{ mobile: 1, tablet: 2 }}
+  itemRows={3}
+  rows={{ mobile: 4, tablet: 2 }}
```

Comment:
Please update the `rows` values according to my comment in the `MatrixResponsiveColumnsGap` demo.